corehq/apps/hqadmin/views/users.py

```diff
@@ -339,7 +339,7 @@ def _validate_user_access(self, user):
 @require_superuser
 def web_user_lookup(request):
     template = "hqadmin/web_user_lookup.html"
-    web_user_email = request.GET.get("q")
+    web_user_email = request.GET.get("q").lower()
```

Comment:
Will this fail if there is no `q`? 

```suggestion
    web_user_email = request.GET.get("q", "").lower()
```

Reply:
`q` always exists, it's a blank string if the user didn't provide it. But your code is more defensive, I like it.

Reply:
:shield: :crossed_swords: